src/Microsoft.DotNet.Cli.Utils/FileAccessRetryer.cs

```diff
@@ -47,5 +48,34 @@ public static class FileAccessRetrier
                 }
             }
         }
+
+        /// <summary>
+        /// Run Directory.Move and File.Move in Windows has a chance to get IOException with HResult 0x80070005. But this error is transient.
+        /// </summary>
+        internal static void RetryOnMoveAccessFailure(Action action)
+        {
+            const int ERROR_HRESULT_ACCESS_DENIED = unchecked((int)0x80070005);
+            int nextWaitTime = 1000;
```

Comment:
Why not start with something like 10ms? That, combined with a 2x back-off factor will give us ~10s to give 10 tries.